spec/controllers/admin/users_controller_spec.rb

```diff
@@ -2,89 +2,115 @@
 
 RSpec.describe Admin::UsersController, type: :controller do
   before do
+    allow(controller).to receive(:authorize) { authorized? }
+    allow(controller).to receive(:pundit_policy_authorized?) { authorized? }
+
     sign_in(user)
   end
 
-  let(:user) { build_stubbed(:user) }
+  let(:user) { build_stubbed(:user, :moderator) }
```

Comment:
@LucasAnderson1984 The role should be irrelevant since you're mocking the controller authorization methods.